x-pack/plugins/security_solution/public/detection_engine/rule_management/logic/bulk_actions/translations.ts

```diff
@@ -70,12 +70,12 @@ export function explainBulkEditSuccess(
         x.type === BulkActionEditType.delete_index_patterns
     )
   ) {
-    return `${i18n.RULES_BULK_EDIT_SUCCESS_DESCRIPTION(summary.succeeded)}. ${
-      i18n.RULES_BULK_EDIT_SUCCESS_INDEX_EDIT_DESCRIPTION
+    return `${i18n.RULES_BULK_EDIT_SUCCESS_DESCRIPTION(summary.succeeded, summary.skipped)} ${
+      summary.skipped > 0 ? i18n.RULES_BULK_EDIT_SUCCESS_DATA_VIEW_RULES_SKIPPED_DETAIL : ''
```

Comment:
Only add the explanation about rules with data views being ignored if the action is index_pattern related, and the API call responded with skipped rules.